mockiato-codegen/src/generate/constant.rs

```diff
@@ -20,17 +19,13 @@ pub(super) fn arguments_lifetime() -> Lifetime {
 }
 
 /// Generates the mock identifier
-///
-/// The name_attr is used if it is supplied
-pub(super) fn mock_struct_ident(trait_decl: &TraitDecl, name_attr: Option<NameAttr>) -> Ident {
+pub(super) fn mock_struct_ident(trait_decl: &TraitDecl) -> Ident {
     const IDENTIFIER_POSTFIX: &str = "Mock";
 
-    name_attr.map(|attr| attr.ident).unwrap_or_else(|| {
-        Ident::new(
-            &format!("{}{}", trait_decl.ident, IDENTIFIER_POSTFIX),
-            trait_decl.span.into(),
-        )
-    })
+    Ident::new(
+        &format!("{}{}", trait_decl.ident, IDENTIFIER_POSTFIX),
```

Comment:
A "postfix" is usually called a "suffix"

Reply:
That's not an issue with my new code, but rather an existing issue that affects (most) code in this file.
I created an issue: https://github.com/myelin-ai/mockiato/issues/82